src/main/java/tools/jackson/databind/JsonNode.java

```diff
@@ -467,6 +467,21 @@ public final boolean isBinary() {
         return getNodeType() == JsonNodeType.BINARY;
     }
 
+    /**
+     * Method that can be used to check whether this node is a numeric
+     * node ({@link #isNumber} would return true)
+     * AND can be converted without loss to it (that is, its value fits
+     * within Java's 16-bit signed integer type, <code>short</code> and
+     * if it is a floating-point number, it does not have fractional part).
+     *<p>
+     * NOTE: this method does not consider possible value type conversion
+     * from non-number types like JSON String into Number; so even if this method returns false,
+     * it is possible that {@link #asShort} could still succeed.
+     *
+     * @since 2.0
```

Comment:
Let's remove this (anything added in 3.0 need not `@since` tags since assumption for new major version -- incompatible with previous ones -- is that anything without tag was in the first minor, 3.0)